docs/compilation/common_errors.md

```diff
@@ -0,0 +1,81 @@
+# Common errors
+
+In this document, we list the most common errors, and mention how the user can fix them.
+
+## 1. Could not find a version that satisfies the requirement concrete-python (from versions: none)
+
+**Error message**: `Could not find a version that satisfies the requirement concrete-python (from versions: none)`
+
+**Cause**: The installation does not work fine for you.
+
+**Possible solutions**:
+- Be sure that you use a supported Python version (currently from 3.8 to 3.11, included)
+- Check you have done `pip install -U pip wheel setuptools` before
+- Consider adding a `--extra-index-url https://pypi.zama.ai`
```

Comment:
now we have https://pypi.zama.ai/cpu or https://pypi.zama.ai/gpu, so users need to select one of those

Reply:
Thx, I did https://github.com/zama-ai/concrete-internal/issues/770
If we need to change it, it would be very soon, because 2.7 is out (but not advertised)